libcaf_core/tests/legacy/mixin/requester.cpp

```diff
@@ -363,4 +363,23 @@ TEST_CASE("GH-1299 regression blocking") {
   }
 }
 
+TEST_CASE("GH-698 regression responding to a request drops the strong "
+          "reference count") {
+  auto weak_client_ref = [this] {
+    return actor_cast<weak_actor_ptr>(
+      sys.spawn([this](event_based_actor* self) {
+        self->request(adding_server, std::chrono::seconds{10}, 1, 2)
```

Comment:
```suggestion
        self->request(adding_server, 10s, 1, 2)
```

(plus `using namespace std::literals`)